packages/terra-date-picker/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 * Changed
   * Visual focus changed to dashed lines for individual fields.
   * Fixed the date format being read 3 times when using SR
+  * Updated screen response to announce when invalid characters 
+    are entered.
+  * Updated screen response to announce when calendar popup dates 
+    are selected.
```

Comment:
Move this to unreleased section

Reply:
https://github.com/cerner/terra-framework/pull/1774/commits/657893f07bc56471de199d189c83edd40a751f4d